website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -38,9 +38,13 @@ Otherwise, all Connect services can talk to all other services.
 The Kubernetes integration takes care of registering Kubernetes services with Consul, injecting a sidecar proxy, and
 enabling traffic redirection.
 
+## Upgrading to Transparent Proxy
+
+When upgrading from older versions (i.e Consul-k8s < `0.26.0` Consul-helm < `0.32.0`) to Consul-k8s >= `0.26.0` and Consul-helm >= `0.32.0`, please make sure to follow the upgrade steps [here](/docs/upgrading/upgrade-specific/#transparent-proxy-on-kubernetes).
 
 ## Configuration
 
+### Enabling Transparent Proxy
```

Comment:
Does it make sense to add a section about setting the annotation on the Namespace and also the order of precedence ?

Reply:
Maybe this belongs in a different page and we just link to it from here? I thought about adding the same info into the health checks page, but not sure where it belongs belongs.

Reply:
I think it does, but maybe this belongs in #10415?

Reply:
Yes good idea, I'll add it in #10415

Reply:
Added to #10415 ✅